docs/html/user_guide.rst

```diff
@@ -324,6 +324,23 @@ To install directly from a wheel archive:
 
       py -m pip install SomePackage-1.0-py2.py3-none-any.whl
 
+.. warning::
+
+   To include optional dependencies provided in the ``provides_extras``
+   metadata in the wheel, you must add quotes around the install target
+   name:
+   
+   .. tab:: Unix/macOS
+
+      .. code-block:: shell
+
+         python -m pip install 'SomePackage-1.0-py2.py3-none-any.whl[my-extras]'
+
+   .. tab:: Windows
+
+      .. code-block:: shell
+
+         py -m pip install 'SomePackage-1.0-py2.py3-none-any.whl[my-extras]'
```

Comment:
IMO it’s enough to make this a regular sentence. It’s not a warning in any case.

Also I would prefer using an explicit relative path, e.g. `./somepackage-...` instead, but that’s just personal preference.

Futhermore, I don’t believe this is a valid wheel name. The wheel specification requires using lowercases.

Reply:
@uranusjr Good points. I've made some changes based on these.

Reply:
@underyx The existing docs already have some usage of not using `./` and also have upper case letters. Should those also be changed in this PR?